pkg/resources/fluentbit/daemonset.go

```diff
@@ -121,6 +122,17 @@ func (r *Reconciler) generateVolumeMounts() (v []corev1.VolumeMount) {
 			MountPath: "/var/log/",
 		},
 	}
+
+	for vCount, vMnt := range r.Logging.Spec.FluentbitSpec.ExtraVolumeMount {
+		if vMnt.Source != "" && vMnt.Destination != "" {
```

Comment:
these fields shouldn't be empty according to the validation pattern

Reply:
done